bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -4,6 +4,78 @@ metadata:
   annotations:
     alm-examples: |-
       [
+        {
```

Comment:
I think these might need to be added in the MGC repo? I think this will get nuked when we do a new bundle as it is pulled in from MGC

Reply:
Right, here I just added the sample files manually to the [kuadrant-operator/config/samples/kustomization.yaml ](https://github.com/Kuadrant/kuadrant-operator/blob/csv-crd-desc/config/samples/kustomization.yaml )from where they propagate to the examples in CSV. 

Fetching these samples automatically from the MGC repo would make more sense I guess. I need to check how to do that in Kustomize. If not possible than I think the samples would need to be fetched in the `make bundle` script.

Reply:
ah I see. Yeah I mean I guess this is ok, but would be better to have them from the source

Reply:
hm. Ok, did some experimentation and looks like we could put a kustomization.yaml to have a remote base with the samples in the MGC repo. The samples for TLSPolicy, ManagedZone and DNSPolicy are already there.
And since the DNSHealthCheckProbe and DNSRecord are created by the controller I suppose there is no real need to expose examples for these two in the Operator Hub through the CSV?

The snipped below would go to: `multicluster-gateway-controller/config/samples/kustomization.yaml`:
```yaml
resources:
- kuadrant.io_v1alpha1_tlspolicy.yaml
- kuadrant.io_v1alpha1_managedzone.yaml
- dnspolicy.yaml
```

Then we could have a reference to it as a remote resource for kustomize in kuadrant-operator:
```yaml
resources:
- github.com/Kuadrant/multicluster-gateway-controller/config/samples?ref=main
```

This would let us keep the samples in one place and not worry about updating them manually in kuadrant-operator.

Reply:
sounds like a good option. If you open the PR I will take a look at it

Reply:
PR opened. I'll leave the link here as a cross reference: [Add config/samples/kustomization.yaml #720](https://github.com/Kuadrant/multicluster-gateway-controller/pull/720)